Tests/pairwise2_testCases.py

```diff
@@ -77,6 +77,23 @@ def test_warnings(self):
             self.assertIn("should not", str(w[-1].message))
 
 
+class TestPairwiseKeywordUsage(unittest.TestCase):
+    """Tests for keyword usage."""
+
+    def test_keywords(self):
+        """Test equality of calls with and without keywords."""
+        aligns = pairwise2.align.globalxx("GAACT", "GAT")
+        aligns_kw = pairwise2.align.globalxx(sequenceA="GAACT",
+                                             sequenceB="GAT")
+        self.assertEqual(aligns, aligns_kw)
+
+        aligns = pairwise2.align.globalmx("GAACT", "GAT", 5, -4)
+        aligns_kw = pairwise2.align.globalmx(sequenceA="GAACT",
+                                             sequenceB="GAT",
+                                             match=5, mismatch=-4)
```

Comment:
The black line breaking would be quite different here - but can leave that till another day.

Reply:
Leave it for `black`, later.